### PR TITLE
Update select-member-groups.png

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the match the new settings

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1360

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger10** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/fundamentals/scope-tags.md [hunk 1]
```diff
@@ -57,15 +57,14 @@ The default scope tag feature is similar to the security scopes feature in Micro
 
 1. In the [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), choose **Tenant administration** > **Roles** > **All roles** > choose a role > **Assignments** > **Assign**.
 2. On the **Basics** page, provide an **Assignment name** and **Description**. Choose **Next**.
-3. On the **Admin Groups** page, choose **Select groups to include**, and select the groups that you want as part of this assignment. Users in these group will have permissions to manage users/devices in the Scope (Groups). Choose **Next**.
+3. On the **Admin Groups** page, choose **Add groups**, and select the groups that you want as part of this assignment. Users in these group will have permissions to manage users/devices in the Scope (Groups). Choose **Next**.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/scope-tags.md:60]:
```suggestion
3. On the **Admin Groups** page, choose **Add groups**, and select the groups that you want as part of this assignment. Users in these groups will have permissions to manage users/devices in the Scope (Groups). Choose **Next**.
```

File: memdocs/intune/fundamentals/scope-tags.md [hunk 2]
```diff
@@ -57,15 +57,14 @@ The default scope tag feature is similar to the security scopes feature in Micro
...
 
     ![Screenshot of select member groups.](./media/scope-tags/select-member-groups.png)
 
-4. On the **Scope Groups** page, select one of the following options for **Assign to**
-    - **Selected groups**: select the groups containing the users/deivces that you want to manage. All users/devices in the selected groups will be managed by the users in the Admin Groups.
-    - **All users**: All users can be managed by the users in the Admin Groups.
-    - **All devices**: All devices can be managed by the users in the Admin Groups.
-    - **All users and all devices**: All users and devices can be managed by the users in the Admin Groups.
+4. On the **Scope Groups** page, select one of the following options for **Included rgroups**
+    - **Add groups**: select the groups containing the users/deivces that you want to manage. All users/devices in the selected groups will be managed by the users in the Admin Groups.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/scope-tags.md:65]:
```suggestion
4. On the **Scope Groups** page, select one of the following options for **Included groups**:
    - **Add groups**: Select the groups containing the users/devices that you want to manage. All users/devices in the selected groups will be managed by the users in the Admin Groups.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Smritib17 — APPROVED** [review]

**Comment by Smritib17** [comment]
#sign-off

**Pull request merged**